resources/views/admin.blade.php

```diff
@@ -367,6 +408,13 @@ function storeEmployee(){
                     $('#company-branch').val($(this).find(':selected').data('branch'))
                 });
 
+                $('#dt-length-0').change(function() {
```

Comment:
Please explain this code

Reply:
the above code handles the misalignment cause the appearance of scroll bar by changing the width the header... @Raza9798 

![image](https://github.com/user-attachments/assets/26414bd0-fcd4-44f2-ae43-3b47ef5b21b1)